test/framework/integration_framework/iroha_instance.hpp

```diff
@@ -47,6 +47,8 @@ namespace integration_framework {
                   const std::string &block_store_path,
                   const std::string &listen_ip,
                   size_t torii_port,
+                  size_t torii_tls_port,
```

Comment:
Look at my comment in `integration_test_framework.cpp` about possible `struct TlsConfig`;

Reply:
Newly introduced arguments are not documented in a docstring above.

Reply:
Fixed in latest version